app/setup/options.js

```diff
@@ -12,6 +13,12 @@ import './lib/ls.attrchange'; // lazy sizes attribute change observer
 
 import { I18N_FALLBACK, I18N_FALLBACK_MESSAGES } from './i18n';
 
+/**
+ * Polyfill for css vars on IE11 https://github.com/jhildenbiddle/css-vars-ponyfill
+ */
+cssVars({
+  // ...
```

Comment:
- [ ] Please enable `silent` in production: https://github.com/jhildenbiddle/css-vars-ponyfill#optionssilent